test_scripts/Polices/user_consent_of_Policies/ATF_isPermissionsConsentNeeded_false.lua

```diff
@@ -0,0 +1,52 @@
+---------------------------------------------------------------------------------------------
+-- Description: 
+--     SDL receives request for app activation from HMI and LocalPT contains permission that don't require User`s consent
+--     1. Used preconditions:
+--			Delete SDL log file and policy table
+-- 			
+--     2. Performed steps
+-- 			Activate with default permissions
+--
+-- Requirement summary: 
+--    [Policies] SDL.ActivateApp from HMI and 'isPermissionsConsentNeeded' parameter in the response
+--
+-- Expected result:
+--      On receiving SDL.ActivateApp PoliciesManager must respond with "isPermissionsConsentNeeded:false" to HMI, consent for custom permissions is not appeared
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+local mobile_session = require('mobile_session')
```

Comment:
@aarotar please avoid unused variables (mobile_session variable isn't used in script)